### PR TITLE
set max image upload to 2mb and use isPending for mutations

### PR DESCRIPTION
**Pull request by TomRadford**

### CODE REVIEW

File: src/components/Export/ExportModal.svelte [hunk 1]
```diff
@@ -164,7 +164,7 @@
   <div slot="body" class="flex flex-col gap-3 px-6 text-regular text-neutral-700">
     {#if $configQuery.isLoading}
       <Loader />
-    {:else if ($exportMutation.isSuccess || $exportMutation.isLoading) && !$exportByIdQuery.isError}
+    {:else if ($exportMutation.isSuccess || $exportMutation.isPending) && !$exportByIdQuery.isError}
```
Comment (martins2s) [inline src/components/Export/ExportModal.svelte:167]: Just for context, changing isLoading to isPending is unrelated to the max image size issue?

Comment (TomRadford) [reply src/components/Export/ExportModal.svelte:167]: unrelated, forgot to change in #478

File: src/components/Import/FileSelector.svelte [hunk 2]
```diff
@@ -12,7 +12,7 @@
   export let selectedFile: File | undefined = undefined;
   export let selectedFiles: File[] = [];
   export let allowMultiple = false;
-  export let maxSize = 5000;
+  export let maxSize = 1024 * 5;
```
[the remark this answers is not in the document]
Comment (TomRadford) [reply src/components/Import/FileSelector.svelte]: 57a7fcf5ba793e326d162e4433566b9edde10c9a

**Review by martins2s — APPROVED** [review]
Nice!

**Pull request merged**